### PR TITLE
feat(plasma-ui): [Product Card] Processing price periodicity

### PR DESCRIPTION
**Pull request by Yakutoc**
Добавлена возможность указывать в цене регулярность платежа.

:baby_chick: Download canary assets:

[colors--canary.229.3540997915.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.229.3540997915.xml)  
[color_sbermarket_ios-swift--canary.229.3540997915.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.229.3540997915.swift)  
[color_sbermarket_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.229.3540997915.kt)  
[color_sbermarket_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.229.3540997915.ts)  
[color_sberprime_ios-swift--canary.229.3540997915.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.229.3540997915.swift)  
[color_sberprime_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.229.3540997915.kt)  
[color_sberprime_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.229.3540997915.ts)  
[PlasmaTokensColor--canary.229.3540997915.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.229.3540997915.swift)  
[typo_mage_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.229.3540997915.kt)  
[typo_mage_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.229.3540997915.ts)  
[typo_plasma_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.229.3540997915.kt)  
[typo_plasma_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.229.3540997915.ts)  
[typo_ruler_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.229.3540997915.kt)  
[typo_ruler_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.229.3540997915.ts)  
[typo_sage_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.229.3540997915.kt)  
[typo_sage_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.229.3540997915.ts)  
[typo_sbermarket_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.229.3540997915.kt)  
[typo_sbermarket_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.229.3540997915.ts)  
[typo_soulmate_kotlin--canary.229.3540997915.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.229.3540997915.kt)  
[typo_soulmate_react-native--canary.229.3540997915.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.229.3540997915.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.114.0-canary.229.3540997915.0
  npm install @salutejs/plasma-ui@1.147.0-canary.229.3540997915.0
  # or 
  yarn add @salutejs/plasma-temple@1.114.0-canary.229.3540997915.0
  yarn add @salutejs/plasma-ui@1.147.0-canary.229.3540997915.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-229/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-229/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-229/
  ui storybook: http://plasma.sberdevices.ru/pr-229/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-229/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-229/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-229/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-229/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/temple-storybook/

  </details>

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-ui/src/components/ProductCard/types.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    readonly?: boolean;
+    /**
+     * Отключить экшен кнопки.
+     */
+    disabledAction?: boolean;
+}
+
+export interface ProductCardPrice extends Pick<ProductCardProps, 'price' | 'oldPrice' | 'periodicity'> {}
+
+export enum PriceTypes {
```
Comment (neretin-trike) [inline packages/plasma-ui/src/components/ProductCard/types.ts:75]: Давай тут const добавим

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-229/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-229/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-229/
  ui storybook: http://plasma.sberdevices.ru/pr-229/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-229/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-229/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-229/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-229/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-229/temple-storybook/

  </details>

**Pull request merged**